src/test/kotlin/com/terraformation/backend/seedbank/model/accession/AccessionModelViabilityTest.kt

```diff
@@ -303,6 +312,44 @@ internal class AccessionModelViabilityTest : AccessionModelTest() {
     Assertions.assertEquals(withdrawnByUserId, model.withdrawals[0].withdrawnByUserId)
   }
 
+  @Test
+  fun `change to seeds tested causes withdrawal and accession remaining seeds to update`() {
+    val initialTest = viabilityTest(seedsTested = 1, startDate = null)
+    val initial =
+        accession()
+            .copy(isManualState = true, remaining = seeds(100))
+            .withCalculatedValues(clock)
+            .addViabilityTest(initialTest, clock)
+
+    Assertions.assertEquals(seeds(99), initial.remaining, "Initial quantity")
```

Comment:
Using `Assertions.` here because the rest of the file uses it; I'll switch to a static import as a separate PR but didn't want to clutter this one with that since it's not related to the bugfix.